config/nightwatch.js

```diff
@@ -47,7 +45,12 @@ module.exports = {
         log_path: './logs/selenium',
         host: '127.0.0.1',
         port: selenium_server_port,
-      }
+      },
+      cli_args: { },
+      test_workers: {
+        enabled: true,
+        workers: 10,
```

Comment:
I saw better responses discarding auto, which I believe defaults to the number of cpu's on the system. I'm guessing that the waits, pauses, and polling for elements to appear prevents the process from becoming cpu-bound.